### ISSUE TITLE
pretrained backbones

### ISSUE THREAD

**Issue by ydhongHIT** [czczup/ViT-Adapter#19]
Thanks for the great work! Did you try to use the imagenet-1k 384x384 finetuned backbones?

**Comment by czczup** [comment]
> Thanks for the great work! Did you try to use the imagenet-1k 384x384 finetuned backbones?

Hi, DeiT provides the [DeiT-base 384](https://dl.fbaipublicfiles.com/deit/deit_base_patch16_384-8de9b5d1.pth) pretrained weights, it can bring slightly gains than the 224 pretrained weights, for example, about ~0.1-0.2 AP for object detection.

**Comment by ydhongHIT** [comment]
> > Thanks for the great work! Did you try to use the imagenet-1k 384x384 finetuned backbones?
> 
> Hi, DeiT provides the [DeiT-base 384](https://dl.fbaipublicfiles.com/deit/deit_base_patch16_384-8de9b5d1.pth) pretrained weights, it can bring slightly gains than the 224 pretrained weights, for example, about ~0.1-0.2 AP for object detection.

How about this pretrained backbone https://conversationhub.blob.core.windows.net/beit-share-public/beit/beit_large_patch16_384_pt22k_ft22kto1k.pth ?

**Comment by czczup** [comment]
> > > Thanks for the great work! Did you try to use the imagenet-1k 384x384 finetuned backbones?
> > 
> > 
> > Hi, DeiT provides the [DeiT-base 384](https://dl.fbaipublicfiles.com/deit/deit_base_patch16_384-8de9b5d1.pth) pretrained weights, it can bring slightly gains than the 224 pretrained weights, for example, about ~0.1-0.2 AP for object detection.
> 
> How about this pretrained backbone https://conversationhub.blob.core.windows.net/beit-share-public/beit/beit_large_patch16_384_pt22k_ft22kto1k.pth ?

I haven't tried this pre-trained weights, since the official repo of BEiT recommends using the weights of pt22k_ft22k. As a similar reference, [MAE-pretrain](https://dl.fbaipublicfiles.com/mae/pretrain/mae_pretrain_vit_base.pth) is slightly better than [MAE-finetune](https://dl.fbaipublicfiles.com/mae/finetune/mae_finetuned_vit_base.pth) in my experiments.

**Comment by ydhongHIT** [comment]
> > > > Thanks for the great work! Did you try to use the imagenet-1k 384x384 finetuned backbones?
> > > 
> > > 
> > > Hi, DeiT provides the [DeiT-base 384](https://dl.fbaipublicfiles.com/deit/deit_base_patch16_384-8de9b5d1.pth) pretrained weights, it can bring slightly gains than the 224 pretrained weights, for example, about ~0.1-0.2 AP for object detection.
> > 
> > 
> > How about this pretrained backbone https://conversationhub.blob.core.windows.net/beit-share-public/beit/beit_large_patch16_384_pt22k_ft22kto1k.pth ?
> 
> I haven't tried this pre-trained weights, since the official repo of BEiT recommends using the weights of pt22k_ft22k. As a similar reference, [MAE-pretrain](https://dl.fbaipublicfiles.com/mae/pretrain/mae_pretrain_vit_base.pth) is slightly better than [MAE-finetune](https://dl.fbaipublicfiles.com/mae/finetune/mae_finetuned_vit_base.pth) in my experiments.

Which task is it? I have seen a paper which claims that 384x384 finetuned backbones achieves higher mIoU on ade20k.

**Comment by czczup** [comment]
> > > > > Thanks for the great work! Did you try to use the imagenet-1k 384x384 finetuned backbones?
> > > > 
> > > > 
> > > > Hi, DeiT provides the [DeiT-base 384](https://dl.fbaipublicfiles.com/deit/deit_base_patch16_384-8de9b5d1.pth) pretrained weights, it can bring slightly gains than the 224 pretrained weights, for example, about ~0.1-0.2 AP for object detection.
> > > 
> > > 
> > > How about this pretrained backbone https://conversationhub.blob.core.windows.net/beit-share-public/beit/beit_large_patch16_384_pt22k_ft22kto1k.pth ?
> > 
> > 
> > I haven't tried this pre-trained weights, since the official repo of BEiT recommends using the weights of pt22k_ft22k. As a similar reference, [MAE-pretrain](https://dl.fbaipublicfiles.com/mae/pretrain/mae_pretrain_vit_base.pth) is slightly better than [MAE-finetune](https://dl.fbaipublicfiles.com/mae/finetune/mae_finetuned_vit_base.pth) in my experiments.
> 
> Which task is it? I have seen a paper which claims that 384x384 finetuned backbones achieves higher mIoU on ade20k.

I only try it on object detection and instance segmentation (with mask rcnn). But on Ade20K, the DeiT-384 pretrain does perform better than DeiT-224.

**Comment by ydhongHIT** [comment]
> > > > > > Thanks for the great work! Did you try to use the imagenet-1k 384x384 finetuned backbones?
> > > > > 
> > > > > 
> > > > > Hi, DeiT provides the [DeiT-base 384](https://dl.fbaipublicfiles.com/deit/deit_base_patch16_384-8de9b5d1.pth) pretrained weights, it can bring slightly gains than the 224 pretrained weights, for example, about ~0.1-0.2 AP for object detection.
> > > > 
> > > > 
> > > > How about this pretrained backbone https://conversationhub.blob.core.windows.net/beit-share-public/beit/beit_large_patch16_384_pt22k_ft22kto1k.pth ?
> > > 
> > > 
> > > I haven't tried this pre-trained weights, since the official repo of BEiT recommends using the weights of pt22k_ft22k. As a similar reference, [MAE-pretrain](https://dl.fbaipublicfiles.com/mae/pretrain/mae_pretrain_vit_base.pth) is slightly better than [MAE-finetune](https://dl.fbaipublicfiles.com/mae/finetune/mae_finetuned_vit_base.pth) in my experiments.
> > 
> > 
> > Which task is it? I have seen a paper which claims that 384x384 finetuned backbones achieves higher mIoU on ade20k.
> 
> I only try it on object detection and instance segmentation (with mask rcnn). This is probably because the MAE-pretrain and MAE-finetune are both 224x224. But on Ade20K, the DeiT-384 pretrain does perform better than DeiT-224 (about 0.5 mIoU).

Thanks for your reply. How about the variations on ADE20K with different seeds? Did you report the best mIoU among them?

**Comment by duanduanduanyuchen** [comment]
> 

We used the default setting of random seed in mmsegmentation without adjustment.
